hexrd/instrument/cylindrical_detector.py

```diff
@@ -283,34 +289,6 @@ def angle_extent(self):
         sz = self.physical_size[1]
         return sz / self.radius / 2.0
 
-    @property
-    def pixel_solid_angles(self):
-        kwargs = {
-            'rows': self.rows,
-            'cols': self.cols,
-            'pixel_size_row': self.pixel_size_row,
-            'pixel_size_col': self.pixel_size_col,
-            'caxis': self.caxis,
-            'paxis': self.paxis,
-            'radius': self.radius,
-            'tvec': self.tvec,
-            'max_workers': self.max_workers,
-        }
-        return _pixel_solid_angles(**kwargs)
-
-    @staticmethod
-    def update_memoization_sizes(all_panels):
-        Detector.update_memoization_sizes(all_panels)
-
-        num_matches = sum(isinstance(x, CylindricalDetector) for x in all_panels)
-        funcs = [
-            _pixel_angles,
-            _pixel_tth_gradient,
-            _pixel_eta_gradient,
-            _pixel_solid_angles,
-        ]
-        Detector.increase_memoization_sizes(funcs, num_matches)
```

Comment:
We still need to keep this function, and just remove `_pixel_solid_angles` from the list. I'm doing that in a commit I am adding.